grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -228,37 +228,47 @@ class CycleCountService {
     }
 
     /**
-     * Batch method to start a cycle count.
-     * Fetch associated Cycle count with the CycleCountRequest and do the following:
-     * if a cycle count doesn't exist, create it, fetch all bin locations + inventory items for the product in a request,
-     * get the product availability for each item, filter out items that have qoh = 0 (but keep those that have qoh = 0 and were included in the previous count)
-     *
-     * */
+     * Batch start multiple cycle counts.
+     */
     List<CycleCountDto> startCycleCount(CycleCountStartBatchCommand command) {
```

Comment:
I did a bit of cleanup on the start cycle count API while I was reworking the product availability flow. Mainly I removed some outdated validation (the updateCycleCount method is not needed anymore) and moved other validation logic to the command object instead like we do for recount.